### PR TITLE
Rely on ActiveAdmin::FormBuilder#has_many using object.class.reflect_on_association(name).klass.new to build nested records

### PR DESCRIPTION
**Pull request by faucct**
The gem does not work with `activeadmin` anymore (from version 1.0.0). This solution seems more stable to `activeadmin` internal changes and supports both previous and latest versions.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ gemspec
 group :test do
   gem 'rails', detect_rails_version
   gem 'activeadmin', github: 'activeadmin', ref: 'ba285ea2fc23ade108ed582134cae8c641f910a4'
```
Comment (bolshakov) [inline Gemfile]: Could you bump activeadmin version to reproduce the issue?

Comment (faucct) [reply Gemfile]: I can do it when this is merged - https://github.com/activeadmin/activeadmin/pull/5253.

Comment (faucct) [reply Gemfile]: Obnovil

**Comment by faucct** [comment]
ping

**Review by bolshakov — APPROVED** [review]
Спасибо, Никита! Отличная работа! 👍

**Comment by faucct** [comment]
Бампни версию, пожалуйста.

**Comment by bolshakov** [comment]
@faucct done

**Pull request merged**